packages/clerk-js/src/core/clerk.ts

```diff
@@ -256,6 +256,7 @@ export default class Clerk implements ClerkInterface {
 
       const { frontendApi, instanceType } = publishableKey as PublishableKey;
 
+      this.publishableKey = key;
```

Comment:
This is specified as a public readonly property, but we weren't setting it anywhere.

Reply:
I'm using it in another spot in this PR to check for a new key, but let me know if we don't want to set this for some reason!